packages/lib/src/h5web/vis-packs/core/line/GlyphMaterial.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
 
 function GlyphMaterial(props: Props) {
   const { color, size, glyphURL } = props;
-  const sprite = useLoader(TextureLoader, glyphURL || GLYPH_URLS.Cross);
+  const sprite = useLoader(TextureLoader, glyphURL ?? GLYPH_URLS.Cross);
```

Comment:
Hm, I wonder if `||` isn't better in this case. 

I mean, setting `glyphURL` to `""` will make this crash with `??` while it will fallback to `Cross` with `||`, right ?

Reply:
Yeah but that's true with any string that is not a valid URL or doesn't correctly encode a resource. The goal here is not to prevent misuse, but to support not passing a glyph URL at all.

Reply:
I see your point, makes sense